src/Nethermind/Nethermind.Db.Rocks/RocksDbExtensions.cs

```diff
@@ -38,7 +38,7 @@ internal static unsafe Span<byte> GetSpan(this RocksDb db, byte[] key, ColumnFam
             throw new RocksDbException(error);
 
         if (result == IntPtr.Zero)
-            return null;
+            return default;
```

Comment:
will this return empty span or null?

Reply:
Span itself can never be null as is ref struct; both will have ɑ null internal pointer, but returning null will go via the implicit operator from Array to Span then to the Span constructor which then you need the Jit to undo all that work to end up with default. So easier just to return default to begin with.